src/handlers/handlerTag.ts

```diff
@@ -0,0 +1,5 @@
+let handlerTag = 1;
```

Comment:
I'd stick to naming one-function-files with the name of the function

Reply:
renamed